.travis.yml

```diff
@@ -13,10 +13,10 @@ before_script:
   - cd build
 
 script:
-  - ../configure CFLAGS="--coverage -ftest-coverage -fprofile-arcs" LDFLAGS=-lgcov --enable-debug && build-wrapper-linux-x86-64 --out-dir bw-output make all
+  - ../configure CFLAGS="--coverage -ftest-coverage -fprofile-arcs" LDFLAGS=-lgcov --enable-debug && build-wrapper-linux-x86-64 --out-dir bw-output make all || true  # Will always fail on external PRs.
```

Comment:
could we split configure step and build step into two lines, makes it much easier to read. we can insist on configure passing and maybe `|| true` only the build part

Reply:
Good idea!